### PR TITLE
hoverRadius

### PR DESCRIPTION
**Pull request by canoceto**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [ ] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/chart.js/index.d.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@
 //                 Takuya Uehara <https://github.com/indigolain>
 //                 Ricardo Mello <https://github.com/ricardo-mello>
 //                 Ray Nicholus <https://github.com/rnicholus>
+//                 Carlos Anoceto <https://github.com/canoceto>
```
Comment (adripanico) [inline types/chart.js/index.d.ts]: Remove trailing whitespace

**Review by adripanico — CHANGES_REQUESTED** [review]

File: types/chart.js/index.d.ts [hunk 2]
```diff
@@ -601,6 +602,7 @@ declare namespace Chart {
     }) => T;
 
     interface ChartDataSets {
+        hoverRadius?: number;
```
Comment (lekoaf) [inline types/chart.js/index.d.ts]: Alphabetical order please.

Also:
> Provide a URL to documentation or source code which provides context for the suggested changes:

You haven't filled in the template properly.

There is already a `hoverRadius` on `ChartPointOptions`. Are they not the same?

**Comment by canoceto** [comment]
No its not. In my current project I need that chance t modified the grow up animation in bubble chart

Sent from Mail for Windows 10

File: types/chart.js/index.d.ts (hunk 1 above)
Comment (adripanico) [reply types/chart.js/index.d.ts]: This is still breaking the build

File: types/chart.js/index.d.ts [hunk 3]
```diff
@@ -620,6 +621,7 @@ declare namespace Chart {
         hoverBackgroundColor?: ChartColor | ChartColor[] | Scriptable<ChartColor>;
         hoverBorderColor?: ChartColor | ChartColor[] | Scriptable<ChartColor>;
         hoverBorderWidth?: number | number[] | Scriptable<number>;
+        hoverRadius?: number;
```
Comment (adripanico) [inline types/chart.js/index.d.ts]: You didn't provided any link to the part of the documentation/code which explain your changes yet. Please review the PR template and complete it accordingly.

File: types/yog-ral/index.d.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
-// TypeScript Version: 2.3
-
-export * from 'node-ral';
+// Type definitions for yog-ral 0.20
+// Project: https://gitlab.baidu.com/fex/node-ral
+// Definitions by: ssddi456 <https://github.com/ssddi456>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+export * from 'node-ral';
```
Comment (adripanico) [inline types/yog-ral/index.d.ts]: This file does not belong to Chart.js types

File: types/yog-ral/tsconfig.json [hunk 5]
```diff
@@ -1,24 +1,24 @@
...
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
+    "files": [
+        "index.d.ts",
+        "yog-ral-tests.ts"
+    ]
+}
```
Comment (adripanico) [inline types/yog-ral/tsconfig.json]: This file does not belong to Chart.js types

File: types/yog-ral/tslint.json [hunk 6]
```diff
@@ -1,3 +1,3 @@
-{
-    "extends": "dtslint/dt.json"
-}
+{
+    "extends": "dtslint/dt.json"
+}
```
Comment (adripanico) [inline types/yog-ral/tslint.json]: This file does not belong to Chart.js types

File: types/yog-ral/yog-ral-tests.ts [hunk 7]
```diff
@@ -1,73 +1,73 @@
...
+yogRal.Config.clearConf;
+yogRal.Config.getConfNames;
+yogRal.Config.getRawConf;
+yogRal.Config.getUpdateNeededRawConf;
+yogRal.Config.enableUpdate;
+yogRal.Config.disableUpdate;
+yogRal.Config.isAutoUpdateEnabled;
+
+const logger = yogRal.Logger('some');
+logger.debug('test');
```
Comment (adripanico) [inline types/yog-ral/yog-ral-tests.ts]: This file does not belong to Chart.js types

**Review by adripanico — CHANGES_REQUESTED** [review]
Please provide documentation links, fix the trailing whitespace and revert the changes on files outside Chart.js scope.

File: types/chart.js/index.d.ts (hunk 3 above)
Comment (canoceto) [reply types/chart.js/index.d.ts]: I got changes from here
https://www.chartjs.org/docs/latest/charts/bubble.html

File: types/chart.js/index.d.ts (hunk 2 above)
Comment (canoceto) [reply types/chart.js/index.d.ts]: Not its not the same

**Review by adripanico — CHANGES_REQUESTED** [review]
There are changes in files that do not belong to Chart.js types.

File: types/chart.js/index.d.ts (hunk 3 above)
Comment (adripanico) [reply types/chart.js/index.d.ts]: According to the docs, this property is also Scriptable.

**Review by adripanico — CHANGES_REQUESTED** [review]

**Pull request closed without merging**